From d2bcd9c33e29a7e2339fd1608bed2b1d41d7b335 Mon Sep 17 00:00:00 2001
From: Janmejay Singh <singh.janmejay@gmail.com>
Date: Wed, 25 Nov 2015 14:44:08 +0000
Subject: [PATCH 02/11] fixed rulebases that had no new-line after last rule,
 which prevents liblognorm(some versions) from loading the last rule

---
 tests/testsuites/mmnormalize_tokenized.rulebase | 2 +-
 tests/testsuites/mmnormalize_variable.rulebase  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/testsuites/mmnormalize_tokenized.rulebase b/tests/testsuites/mmnormalize_tokenized.rulebase
index da5242d..0b9bd86 100644
--- a/tests/testsuites/mmnormalize_tokenized.rulebase
+++ b/tests/testsuites/mmnormalize_tokenized.rulebase
@@ -2,4 +2,4 @@ rule=only_ips:%only_ips:tokenized:, :ipv4%
 rule=local_ips:local ips are %local_ips:tokenized:, :ipv4%
 rule=external_ips:%external_ips:tokenized:, :ipv4% are external ips
 rule=paths:for %user:char-to:@%@localhost path was %fragments:tokenized:\x3a:char-sep:\x3a%
-rule=recur_comma_colon_nos:comma separated list of colon separated numbers: %some_nos:tokenized:, :tokenized: \x3a :tokenized:#:number%
\ No newline at end of file
+rule=recur_comma_colon_nos:comma separated list of colon separated numbers: %some_nos:tokenized:, :tokenized: \x3a :tokenized:#:number%
diff --git a/tests/testsuites/mmnormalize_variable.rulebase b/tests/testsuites/mmnormalize_variable.rulebase
index 4d40d4c..b386787 100644
--- a/tests/testsuites/mmnormalize_variable.rulebase
+++ b/tests/testsuites/mmnormalize_variable.rulebase
@@ -1 +1 @@
-rule=hms:%hr:number%:%min:number%:%sec:number% %zone:word%
\ No newline at end of file
+rule=hms:%hr:number%:%min:number%:%sec:number% %zone:word%
-- 
2.1.0

